Amun InspectionRun Analysis notebook

pacospace opened the pull request.

Commit:
Amun InspectionRun Analysis notebook

CermakM commented:
Shouldn't we create a separate branch for collaboration purposes? @pacospace WDYT?

Commit:
Set up structure and pair with a .py file

The structure has been set up according to the Trello card: https://trello.com/c/GG6unWnD/531-create-notebook-to-analyse-amun-inspections-nokr

Signed-off-by: Marek Cermak <macermak@redhat.com>

modified:   notebooks/development/Amun InspectionRun Analysis .ipynb
new file:   notebooks/development/Amun InspectionRun Analysis .py

Commit:
Merge pull request #1 from CermakM/new-notebook-AmunInspectionRunAnalysis

Set up structure and pair with a .py file

Commit:
Describe inspection job json structure

Commit:
Profile and process inspection results

The list of inspection results in JSON format has been profiled and
parsed into a pandas DataFrame.

Constant variables have been excluded from the resulting DataFrame.

Signed-off-by: Marek Cermak <macermak@redhat.com>

modified:   Amun InspectionRun Analysis .ipynb
modified:   Amun InspectionRun Analysis .py

Commit:
Merge pull request #2 from CermakM/amun-inspections-data-processing

Profile and process inspection results

Commit:
Modified functions for the description

Commit:
Gather statistics and visualize inspection duration

Signed-off-by: Marek Cermak <macermak@redhat.com>

modified:   Amun InspectionRun Analysis .ipynb
modified:   Amun InspectionRun Analysis .py

Commit:
Merge pull request #3 from CermakM/amun-inspections-duration-statistics

Gather statistics and visualize inspection duration

Commit:
Added little introduction

Commit:
Improved description

Commit:
Provide simple functions to visualize inspections

Signed-off-by: Marek Cermak <macermak@redhat.com>

modified:   notebooks/development/Amun InspectionRun Analysis .py

Commit:
Re-generate notebook and add DataFrame parsing step

Signed-off-by: Marek Cermak <macermak@redhat.com>

modified:   Amun InspectionRun Analysis .ipynb
modified:   Amun InspectionRun Analysis .py

Commit:
Merge pull request #4 from CermakM/amun-inspections-library-usage

Amun inspections library usage

Commit:
Little changes and show some cells outputs
